core/templates/services/site-analytics.service.ts

```diff
@@ -36,7 +36,7 @@ export class SiteAnalyticsService {
   constructor(private windowRef: WindowRef) {}
 
   // For definitions of the various arguments, please see:
-  // developers.google.com/analytics/devguides/collection/analyticsjs/events
+  // developers.google.com/analytics/devguides/collection/analyticsjs/events.
```

Comment:
I thought you didn't want to put periods on the end of URLs? (which I agree with)

Reply:
Ah sorry, I have put it here by mistake. I will remove it